packages/block-library/src/media-text/edit.native.js

```diff
@@ -183,10 +188,10 @@ class MediaTextEdit extends Component {
 					/>
 				</BlockControls>
 				<View style={ containerStyles }>
-					<View style={ { width: widthString } }>
+					<View style={ { width: widthString, ...( isSelected ? { padding: 12 } : { padding: isParentSelected || isAncestorSelected ? 6 : 16 } ) } } >
```

Comment:
we could refactor this into a separate function, for clarity.

Reply:
done.